test/flapjax.test.js

```diff
@@ -11,35 +11,23 @@
 
   function id(x) { return x; }
 
-  describe('Flajax', function() {
+  describe('flapjax', function() {
     it('exists', function() {
       expect(flapjax).to.be.a.function;
     });
 
-    it('takes options and returns a Task', function() {
-      var task = flapjax(maria);
-      expect(task).to.be.an.object;
+    it('takes options and returns a Future', function() {
+      var future = flapjax(maria);
+      expect(future).to.be.an.object;
       ['fork', 'cleanup', 'ap', 'bimap', 'cata', 'chain', 'concat',
        'empty', 'fold', 'map', 'of', 'orElse', 'rejected',
        'rejectedMap', 'swap'].forEach(function(method) {
         expect(task[method]).to.be.a.function;
       });
     });
     
-    it('takes an optional decorator function from XHR-> XHR', function(done) {
-      function dec(xhr) {
-        xhr.withCredentials = true;
-        return xhr;
-      }
-      flapjax(maria, dec).fork(
-        function(x) {
-          expect(x.withCredentials).to.be.true;
-          done();
-        },
-        function(x) {
-          expect(x.withCredentials).to.be.true;
-          done();
-        });
+    it('', function() {
```

Comment:
_mea culpa_. I have been derelict on the testing, it's on the bottom of my priority list